src/services/replyService.js

```diff
@@ -15,36 +15,31 @@ const {
 } = require('../utils/constants');
 
 module.exports = {
-  broadcastAndStoreReply: function(authenticatedUserInstance, { postId, body, createdAt }) {
-    return PostModel.findById(postId)
-      .then(postRecord => {
-        const postAuthor = postRecord.userId;
-        const postPermLink = postRecord.permLink;
-        const permLink = this.replyPermlink(postAuthor, postPermLink,
-          authenticatedUserInstance.username, createdAt)
-        return authenticatedUserInstance.broadcastReply({ postAuthor, postPermLink, permLink, body });
-      })
-      .then(broadcastSuccess => {
-        if (broadcastSuccesss) {
-          return authenticatedUserInstance.userInOurDb;
-        }
-      })
-      .then(userRecord => {
-        const permLink = this.replyPermlink(postAuthor, postPermLink,
+  broadcastAndStoreReply: function(authenticatedUserInstance, postPermLink, postAuthor, body) {
+      const createdAt = new Date();
```

Comment:
We shouldn't have to manually create the `createdAt` date. It's automatically created for us by virtually all db software. Sequelize creates a `created_at`/`createdAt` variable for us. I'm sure creating a user via steem will give us the `created` time var